src/components/general/ErrorMessage/index.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import Button from "../Button";
+import styles from "./styles.less";
+import * as Icons from "./errorIcons";
+import classNames from "classnames";
+
+export enum ErrorTypes {
+    error= "error",
+    accessDenied= "accessDenied",
+    notFound= "notFound",
+    noData= "noData",
+    noAttachment= "noAttachment",
```

Comment:
noAttachment and noActionCreated look like they're specific for meetings/reviews. We shouldn't have app specific error types and messages in a common component. Should be able to use the title and message props to provide that

Reply:
Removed noAttachment and noActionCreated